eng/docgeneration/script/docgen.ps1

```diff
@@ -0,0 +1,61 @@
+﻿# Extract Dlls from nuget package
+# Copy the items to the dll directory
+function ExtractDlls($PkgDirPath, $DllDirPath) {
```

Comment:
We should try and avoid extracting these and instead restore them via a project.

Reply:
Restoring does not generate the .dlls for this, except I build the entire repo which I was trying to avoid.

Reply:
Aren't you extracting binaries from the nuget package? If so you should let nuget do that. You can do your restore against a local set of packages. What I want to try and avoid is you duplicating work that nuget is already good at, we shouldn't hard-code knowledge about the target framework folders and such in these scripts because target frameworks will change overtime but nuget will be able to know the relationship between them and select the correct assets.